README.markdown

```diff
@@ -24,14 +31,19 @@ In BSD:
 	cd bsd && make
 
 Installation
-------------
```

Comment:
why?

Reply:
It's build built and installation from sources. For me it looks like a just a build step. I can revert